From 14639346c5891888c45e886881433bb7be547015 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 21 Feb 2024 10:42:08 +0100
Subject: [PATCH 0611/1695] dt-bindings: add STM32MP21x shared bindings

Adds STM32MP21x SoC family bindings that share STM32MP25
RISAF and RISAB bindings.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git fe9409966b1f8df9ceaf94a41c5cb98b1b866a90 dt-bindings: add STM32MP21x shared bindings]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I7a7d8735d8add8ab32f760715b6ffc8d224d7bf8
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/362584
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/include/drivers/stm32mp_dt_bindings.h | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/core/include/drivers/stm32mp_dt_bindings.h b/core/include/drivers/stm32mp_dt_bindings.h
index ae65a60a6..d0f16b570 100644
--- a/core/include/drivers/stm32mp_dt_bindings.h
+++ b/core/include/drivers/stm32mp_dt_bindings.h
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
 /*
- * Copyright (c) 2017-2022, STMicroelectronics
+ * Copyright (c) 2017-2024, STMicroelectronics
  */
 #ifndef __DRIVERS_STM32MP_DT_BINDINGS_H
 #define __DRIVERS_STM32MP_DT_BINDINGS_H
@@ -35,4 +35,13 @@
 #include <dt-bindings/soc/stm32mp25-risaf.h>
 #endif
 
+#ifdef CFG_STM32MP21
+#include <dt-bindings/clock/st,stm32mp21-rcc.h>
+#include <dt-bindings/clock/stm32mp21-clksrc.h>
+#include <dt-bindings/reset/st,stm32mp21-rcc.h>
+#include <dt-bindings/soc/stm32mp25-rif.h>
+#include <dt-bindings/soc/stm32mp21-rifsc.h>
+#include <dt-bindings/soc/stm32mp25-risaf.h>
+#endif
+
 #endif /* __DRIVERS_STM32MP_DT_BINDINGS_H */
-- 
2.47.2

